include/LightGBM/utils/openmp_wrapper.h

```diff
@@ -66,6 +67,22 @@ class ThreadExceptionHelper {
 
 #else
 
+/*
+ * To be compatible with openmp, define a nothrow macro which is used by gcc
+ * openmp, but not by clang.
+ * See also https://github.com/dmlc/dmlc-core/blob/main/include/dmlc/omp.h#L14
```

Comment:
```suggestion
 * See also https://github.com/dmlc/dmlc-core/blob/3106c1cbdcc9fc9ef3a2c1d2196a7a6f6616c13d/include/dmlc/omp.h#L14
```

Could you please change this to a link to a specific commit? This will make the link understandable even if code gets moved / changed in `dmlc-core`.